src/main/java/frc/robot/commands/MakeIntakeMotorSpin.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.commands;
```

Comment:
Please rename this class to something like "IntakeCommand"

Reply:
ok